toolkit/tools/internal/directory/directory.go

```diff
@@ -28,3 +31,37 @@ func LastModifiedFile(dirPath string) (modTime time.Time, latestFile string, err
 
 	return
 }
+
+// CopyContents will recursively copy the contents of srcDir into dstDir.
+// - It will create dstDir if it does not already exist.
+func CopyContents(srcDir, dstDir string) (err error) {
+	const squashErrors = false
+
+	err = os.MkdirAll(dstDir, os.ModePerm)
+	if err != nil {
+		return
+	}
+
+	fds, err := ioutil.ReadDir(srcDir)
+	if err != nil {
+		return
+	}
+
+	for _, fd := range fds {
+		srcPath := filepath.Join(srcDir, fd.Name())
+		dstPath := filepath.Join(dstDir, fd.Name())
+
+		cpArgs := []string{"-a", srcPath, dstPath}
+		if fd.IsDir() {
+			cpArgs = append([]string{"-r"}, cpArgs...)
+		}
+
+		err = shell.ExecuteLive(squashErrors, "cp", cpArgs...)
+		if err != nil {
+			return
+		}
+	}
```

Comment:
Can we only check if `srcDir` is actually a directory and do `cp -ra srcDir/* dstDir`? Or was this causing some issues?

Reply:
The wildcard in the above `cp` command would need to be handled by bash wildcard expansion. In the case of the go tooling, we directly execute the program "cp" instead of going through bash so that kind of expansion isn't possible.

I'll add a check for `srcDir` being a directory.